frontend/templates/product-list/MetaTags.tsx

```diff
@@ -37,9 +37,11 @@ export function MetaTags({ productList }: MetaTagsProps) {
    }
    title += ' | iFixit';
    const itemTypeHandle = itemType ? `/${encodeDeviceItemType(itemType)}` : '';
-   const canonicalUrl = `${appContext.ifixitOrigin}${
-      productList.path
-   }${itemTypeHandle}${page > 1 ? `?${PRODUCT_LIST_PAGE_PARAM}=${page}` : ''}`;
+   const canonicalUrl = encodeURI(
+      `${appContext.ifixitOrigin}${productList.path}${itemTypeHandle}${
+         page > 1 ? `?${PRODUCT_LIST_PAGE_PARAM}=${page}` : ''
+      }`
```

Comment:
I think I'd rather encodeURIComponent on just the ItemTypeHandle (probably inside encodeDeviceItemType) rather than encoding the whole url. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURI There's a subtle difference between the two functions and in this case it might not matter (at least most of the time) but I don't really trust encoding the whole url.